src/rocksdb_options.rs

```diff
@@ -949,6 +974,10 @@ impl Options {
             crocksdb_ffi::crocksdb_options_set_allow_concurrent_memtable_write(self.inner, v);
         }
     }
+
+    pub fn set_env(&mut self, env: &Env) {
```

Comment:
I think you should keep the Env in the Options. Otherwise, the Env may be destroyed quickly when dropping Env.